README.md

```diff
@@ -1,2 +1,162 @@
+
+Create a user and assign roles
+
+You need to create new users on the sub-organizations with the required permissions.
+To create a user for Best Auto Mart with all administrative permissions :
+
+1. Click Sub Organization at left navigation panel
+2. Use the Organization Switcher to change the organization to Best Car Mart.
+3. Click User Management at the left navigation panel and select Users.
+   <img src="./images/docs/users.png" alt="Users" title="Users">
+
+4. Click Add User button and create a user named Alex on the Best Car Mart organization.
+5. Create an admin role with all permissions.
+6. Click User Management > Roles > Organization Roles> Configure.
+   <img src="./images/docs/organizationRoles.png" alt="organizationRoles" title="organizationRoles">
+
+7. Click Add Organization Role button and create an administrator role.
+8. Click next and check all permissions to assign all the permissions to the created role.
+9. Click next and select the users tab.
+10. Check the checkbox alex and select the front arrow to assign the user to the role.
+    <img src="./images/docs/userRole.png" alt="userRole" title="userRole">
+
+11. create another user named Tom on the Best Auto Mart organization( without assiging roles)
+
+## 🚀 Getting Started
+
+1. Clone the repository.
+
+```bash
+git clone https://github.com/ChanikaRuchini/asgardeo-b2b-nextjs-sample-app.git
+```
+
+2. Install the dependencies.
+
+```bash
+npm install
+```
+
+3. Create a `.env` file based on the `.env.example` file.
+
+```bash
+cp .env.example .env
+```
+
+4. Update the values in the `.env` file based on your requirements.
+
+```bash
+
+# The endpoint of the server's API that the client application will communicate with
+# E.g., http://localhost:3000
+NEXT_PUBLIC_HOSTED_URL="http://localhost:3000"
+
+# The base URL for the Asgardeo root organization's API
+# E.g., https://api.asgardeo.io/t/your-org
+NEXT_PUBLIC_ASGARDEO_BASE_ORGANIZATION_URL="https://api.asgardeo.io/t/guardioinc"
+
+# The client ID for the Asgardeo Shared Application
+SHARED_APP_CLIENT_ID="<CLIENT ID OF CREATED APP>"
+
+# The client Secret for the Asgardeo Shared Application
+SHARED_APP_CLIENT_SECRET="<CLIENT SECRET OF CREATED APP>"
+
+# The name of the Asgardeo Shared Application
+SHARED_APPICATION_NAME="Guardio-App"
+
+# Scopes
+API_SCOPES="openid email profile internal_login internal_user_mgt_view internal_user_mgt_list internal_user_mgt_update internal_user_mgt_delete internal_user_mgt_create internal_idp_view internal_idp_create internal_idp_update internal_idp_delete internal_application_mgt_view internal_application_mgt_update internal_application_mgt_create internal_application_mgt_delete internal_organization_view internal_role_mgt_view internal_role_mgt_create internal_role_mgt_update internal_role_mgt_delete internal_group_mgt_update internal_group_mgt_view internal_group_mgt_create internal_group_mgt_delete internal_governance_view internal_governance_update"
```

Comment:
Do we need all these scopes to run the sample app?

Reply:
Yes